packages/@lightningjs/ui-components/src/types/lui.d.ts

```diff
@@ -34,3 +34,11 @@ export type StylePartial<T> = {
     : // If it is NOT an object, then just spit back the original value type.
       T[P];
 };
+
+// TODO: flag for discussion
+// more specific naming of this object
+export type TransitionObject = {
```

Comment:
Leaving a comment here for more visibility. Any thoughts on what we should be naming this object? Anthony first suggested `TransitionSetting`.

Reply:
From standup, ask Frank if there's already an object that has the transition properties and if so we could use that. From Anthony, if the object has more properties than we need, we can create a subprop that picks only select properties rather than the entire object.

Reply:
Looks like LightningCore already has something called `TransitionSettings`!